docs/dev-guide/conventions/sync-v-async.rst

```diff
@@ -50,3 +52,24 @@ Example Call Report::
   "spawned_tasks": [{"_href": "/pulp/api/v2/tasks/7744e2df-39b9-46f0-bb10-feffa2f7014b/",
                      "task_id": "7744e2df-39b9-46f0-bb10-feffa2f7014b" }]
  }
+
+.. _group_call_report:
+
+Group Call Report
+-----------------
+
+A 202 ACCEPTED response returns a **Group Call Report** JSON object as the response body that has
+the following fields:
+
+* **_href** - Path to the root of task group resource. However, this API endpoint currently
+  returns 404 in all cases. You can append `/state-summary/` to the URL and perform a GET request
```

Comment:
Drop the leading / to stay within the normal "urljoin" concepts, in which a leading / would cause the entire path of the URL to be overwritten.